lib/capistrano/tasks/assets.rake

```diff
@@ -33,7 +33,8 @@ namespace :fos do
         DESC
         task :dump do
             on roles(:app) do
-                execute "cd #{release_path} && php " + fetch(:symfony_console_path) + " --env=prod fos:js-routing:dump --target=web/js/fos_js_routes_prod.js"
+                execute "cd #{release_path} && php " + fetch(:symfony_console_path) + " --env=prod fos:js-routing:dump --target=web/js/fos_js_routes.js"
+                execute "cp web/js/fos_js_routes.js web/js/fos_js_routes_prod.js"
```

Comment:
@jdubuisson I'm wondering if the `cd #{release_path} && ` is required in order to avoid side-effects.

Are we sure that the second line is executed in the same context than the first line? Maybe these commands are executed by separated processes than don't share the same current directory.

Reply:
:+1: better safe than sorry :smiley: